lib/cli/resolve-input.js

```diff
@@ -5,27 +5,41 @@
 const memoizee = require('memoizee');
 const parseArgs = require('./parse-args');
 
-const baseArgsSchema = {
-  boolean: new Set(['version', 'help', 'help-interactive', 'v']),
-  string: new Set(['config']),
-  alias: new Map([
-    ['c', 'config'],
-    ['h', 'help'],
-  ]),
-};
+const customSAliasCommands = new Set(['config credentials'], ['config  tabcompletion install']);
```

Comment:
Should there be two spaces between `config` and `tabcompletion`?

Reply:
Very good point, there shouldn't. Updated